util/gitutil/git_commit.go

```diff
@@ -0,0 +1,19 @@
+package gitutil
+
+func IsCommitSHA(str string) bool {
+	if len(str) != 40 {
```

Comment:
Relevant to https://git-scm.com/docs/hash-function-transition, this should probably _also_ accept 64, right?